src/containerDeployment/containerDeploymentWebviewController.ts

```diff
@@ -147,6 +166,21 @@ export class ContainerDeploymentWebviewController extends FormWebviewController<
             currentStep.loadState = stepSuccessful ? ApiStatus.Loaded : ApiStatus.Error;
             if (stepSuccessful) {
                 state.currentDockerStep += 1; // Move to the next step
+            } else {
+                // If the step failed, log the error and send telemetry
+                // Error telemetry includes the step number and error message
+                sendErrorEvent(
+                    TelemetryViews.ContainerDeployment,
+                    TelemetryActions.RunDockerStep,
+                    new Error(currentStep.errorMessage),
+                    true, // includeErrorMessage
+                    undefined, // errorCode
+                    undefined, // errorType
+                    {
+                        dockerStep: cd.DockerStepOrder[currentStepNumber],
+                        fullErrorMessage: currentStep.fullErrorText || "",

```

Comment:
is this necessary with includeErrorMessage: true on line 176?  Check to see what's already getting emitted